fairlearn/reductions/_exponentiated_gradient/_lagrangian.py

```diff
@@ -28,12 +28,7 @@ def __init__(self, classifier):
         self._classifier = classifier
 
     def __call__(self, X):
-        pred = self._classifier.predict(X)
-        # Some estimators return an output of the shape (num_preds, 1) - flatten such
-        # results
-        if getattr(pred, "flatten", None) is not None:
```

Comment:
Why is this no longer needed?

Reply:
I think this was an artifact of improperly wrapping pytorch classifiers in our tests. scikit-learn output conventions are here:
https://scikit-learn.org/stable/glossary.html#term-predict
The supported shapes are (n_samples,) and (n_samples, n_outputs), but not (n_samples, 1). As far as I can tell, we have never supported (n_samples, n_outputs) convention (meaning one-hot-encoded predictions)--that might be nice to add, but in a separate PR I think.